code/modules/atmospherics/machinery/components/binary_devices/volume_pump.dm

```diff
@@ -70,7 +71,7 @@
 		return
 
 
-	var/transfer_ratio = transfer_rate / air1.volume
```

Comment:
what? why

Reply:
because air1.volume is 200 L. if i change the airs volume to another value the transfer ratio can't be from 0 to 1 anymore. like the max transfer_rate is 200, normally you'd have 200/200=1. with the volume increased to 600 the ratio would change (and since we are using remove_ratio() to move things around it would be bad)

The other way would be to increase the max_transfer_rate to 600